src/main/java/ui/CreateRecipeUI.java

```diff
@@ -24,10 +24,10 @@ public class CreateRecipeUI extends JFrame implements RecipeCreateBox{
     JLabel nameQ = new JLabel("What's the name of the recipe?");
     JLabel procedureQ = new JLabel("What's the procedure of the recipe?");
     JLabel cuisineQ = new JLabel("What's the cuisine type?");
-    JLabel ingredientsQ = new JLabel("What are the ingredients");
-    JLabel caloriesQ = new JLabel("What is the total calories?");
-    JLabel timeQ = new JLabel("What is the total time required?");
-    JLabel difficultyQ = new JLabel("What is the difficulty(out of 5)?");
+    JLabel ingredientsQ = new JLabel("What are the ingredients? (Use comma to separate)");
```

Comment:
should we add "use singular form" here too? I added it in search/sort since it can only find exact matches.